tests/test_cout.py

```diff
@@ -0,0 +1,33 @@
+import sys
+
+import pytest
+
+from cppstream import cout as _real_cout
+
+
+@pytest.fixture
+def cout():
+    _real_cout._stream = sys.stdout
```

Comment:
setup, test run and teardown get different capture objects, so its expected to fail like this

Reply:
Getting and replacing the stream dynamically works now